pkgs/tools/text/asciigraph/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
```

Comment:
Why not use buildGoModule and save the deps.nix?

Reply:
The upstream is sadly not using `go.mod`.